components/experimental/src/dimension/units/format.rs

```diff
@@ -71,6 +71,23 @@ fn test_basic() {
     formatted_unit.write_to(&mut sink).unwrap();
     assert_eq!(sink.as_str(), "12,345.67 m");
 
+    let locale = locale!("en-US").into();
+    let century = "century";
+    let fmt = UnitsFormatter::try_new(
+        &locale,
+        century,
+        UnitsFormatterOptions {
+            width: Width::Long,
+            ..Default::default()
+        },
+    )
+    .unwrap();
+    let value = "12345.67".parse().unwrap();
```

Comment:
let's use a realistic value

Reply:
In testing, it is better to use a number that is obviously random. This ensures that the user or the reviewer can easily detect that the number is merely a placeholder and does not carry any specific meaning. Using such clearly arbitrary numbers prevents confusion and helps focus on the test’s purpose.

Reply:
Exactly, no body said `"12345.67 centuries". `, that is the point, it is a test case and the number does not mean anything.

Reply:
test cases should represent realistic invocations, and it's valuable to use different values for different test cases in order to increase code coverage, but I will not spent any more time arguing this.

Reply:
I agree with you, especially in the document tests, but If I am just testing the flow, i prefer random numbers. 

By the way, this is not part of this PR right now anymore ,